content/source/docs/enterprise/workspaces/variables.html.md

```diff
@@ -68,7 +68,7 @@ To protect these secrets, you can mark any any Terraform or environment variable
 
 Marking a variable as sensitive prevents anybody (including you) from viewing its value in TFE's UI or API.
 
-Users with edit permissions can set new values for sensitive variables. However, no one can change a sensitive variable into a non-sensitive one. (If a variable was accidentally marked sensitive, you can delete it and create a new variable to replace it.)
+Users with edit permissions can set new values for sensitive variables. However, no one can change a sensitive variable's key, hcl flag, nor change a senstive variable into a non-sensitive one. (If a variable was accidentally marked sensitive, you can delete it and create a new variable to replace it.)
 
```

Comment:
@beekus I had the impression that we were changing this so no updating would be possible?

Reply:
we are allowing values to be changed, but nothing else.

Reply:
That is so much clearer. Thanks!